config_example.json

```diff
@@ -28,7 +28,10 @@
             "devedition_candidates_repacks": "/path/to/beetmoverscript/beetmoverscript/templates/devedition_candidates_repacks.yml"
         },
         "push-to-releases": {},
-        "push-to-staging": {}
+        "push-to-partner": {
```

Comment:
As stated above, we'll either need to support another behavior (`push-to-partner-public` vs `push-to-partner-private`?) or we'll need templates that can handle the various partner files.